forte/processors/__init__.py

```diff
@@ -19,3 +19,4 @@
 from forte.processors.vocabulary_processor import *
 from forte.processors.bert_based_query_creator import *
 from forte.processors.lowercaser_processor import *
+from forte.processors.allennlp_processors import *
```

Comment:
Sure, makes sense. I've removed them.
Why are some other processors being imported there then? I got confused by that.

Reply:
General guideline is to not import 3rd party libraries in this init file. If we have third party imports here, we need to remove them.

Reply:
Got it. Should we remove the `ner`, `srl` as they have dependencies mentioned in the `extras_require` in `setup.py`? Not sure about others.

Reply:
Also, from user perspective they'll have to import like 
`from forte.processors.allennlp_processor import AllenNLPProcessor` 
instead of 
`from forte.processors import AllenNLPProcessor`
Is that fine?

Reply:
Yeah, since we don't want the users to install all 3rd party all the time.  A longer import is just a small inconvenience to me. 

NER and SRL are our own models, we hope we can include them in the core. Right now they are using a few 3rd party dependencies, let's see if we can clean them up.

Reply:
Got it, thanks 👍